src/V3EmitCSyms.cpp

```diff
@@ -354,6 +354,7 @@ class EmitCSyms : EmitCBaseVisitor {
         m_ofpBase = NULL;
         iterate(nodep);
     }
+    virtual ~EmitCSyms() VL_OVERRIDE { VL_DO_DANGLING(delete m_ofpBase, m_ofpBase); }
```

Comment:
I take it this was an unrelated leak - please submit a separate pull.

Reply:
I just submitted PR #2254